### ISSUE TITLE
Return the language when using "autodetect" feature

### ISSUE THREAD

**Issue by dpordomingo** [bblfsh/sdk#203]
When it is used "autodetect" feature, passing the `filename` and `content` in the request, it would be great to obtain the `language` as part of the response.

It could be something like:
```protobuf
message ParseResponse {
	option (gogoproto.goproto_getters) = false;
	option (gogoproto.goproto_stringer) = false;
	option (gogoproto.typedecl) = false;
	gopkg.in.bblfsh.sdk.v1.protocol.Status status = 1;
	repeated string errors = 2;
	google.protobuf.Duration elapsed = 3 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
	gopkg.in.bblfsh.sdk.v1.uast.Node uast = 4 [(gogoproto.customname) = "UAST"];
	string language = 5;
}
```

instead of the current [ParseResponse](https://github.com/bblfsh/sdk/blob/master/protocol/generated.proto#L47):
```protobuf
message ParseResponse {
	option (gogoproto.goproto_getters) = false;
	option (gogoproto.goproto_stringer) = false;
	option (gogoproto.typedecl) = false;
	gopkg.in.bblfsh.sdk.v1.protocol.Status status = 1;
	repeated string errors = 2;
	google.protobuf.Duration elapsed = 3 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
	gopkg.in.bblfsh.sdk.v1.uast.Node uast = 4 [(gogoproto.customname) = "UAST"];
}
```

**Comment by juanjux** [comment]
Yes. Also for `NativeParseResponse`.

**Comment by juanjux** [comment]
Fixed by SDK v1.8.0 (already incorporated by latest Python and Java drivers) and bblfshd v2.2.0.
